### PR TITLE
kty 0.3.1 (new formula)

### PR DESCRIPTION
**Pull request by p-linnane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/k/kty.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    depends_on "openssl@3"
+  end
+
+  def install
+    inreplace "Cargo.toml", "0.0.0-UNSTABLE", version.to_s
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match version.to_s, shell_output(bin/"kty -V")
```
Comment (carlocab) [inline Formula/k/kty.rb:22]:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/kty -V")
```
Technically `bin/"kty -V"` is not a `Pathname`. (But `bin/"kty"` is.)

File: Formula/k/kty.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    assert_match "failed to read an incluster environment variable",
+                 shell_output(bin/"kty users check brew 2>&1", 1)
```
Comment (carlocab) [inline Formula/k/kty.rb:25]:
```suggestion
                 shell_output("#{bin}/kty users check brew 2>&1", 1)
```

**Review by carlocab — APPROVED** [review]

File: Formula/k/kty.rb (hunk 1 above)
Comment (carlocab) [inline Formula/k/kty.rb:22]:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/kty -V")
```
Think you dropped this.

Comment (p-linnane) [reply Formula/k/kty.rb]: Thanks. Something went wrong when rebasing. Fixed.

**Review by carlocab — APPROVED** [review]

**Pull request merged**